### PR TITLE
[MISC] Add docstring checks in flake8 configuration

### PR DESCRIPTION
**Pull request by aiola**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ input (``*.in``) and actual (``*.txt``) requirements files for CI and prod envir
 This project uses ``black`` (https://github.com/psf/black) for formatting and enforcing a coding style.
 Execute ``black src`` to reformat all source code files according to PEP 8 specifications.
 
-We use ``flake8`` () for static code analysis. The configuration file is located in the root: ``.flake8``.
+We use ``flake8`` (https://github.com/PyCQA/flake8) for static code analysis. The configuration file is located in the root: ``.flake8``.
```
Comment (NicolaDonelli) [inline README.md:40]: Is this still true? Aren't flake8's configurations in setup.cfg?

**Review by NicolaDonelli — APPROVED** [review]
Just a small typo in README.md (I think), but in general is fine.

**Review by deusebio — CHANGES_REQUESTED** [review]
Looks fine apart from a small (but important) thing... ;)

File: setup.cfg [hunk 2]
```diff
@@ -10,12 +10,20 @@ addopts = --doctest-modules --cov=./tests
...
+    # Ignored to work with Sphinx
+    RST303, RST304, RST307
+
 # line length is intentionally set to 80 here because black uses Bugbear
 # See https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length for more details
 max-line-length = 80
+
 max-complexity = 18
-select = B,C,E,F,W,T4,B9
+select = B,C,D,E,F,W,T4,B9,RST
```
[the remark this answers is not in the document]
Comment (NicolaDonelli) [reply setup.cfg:26]: Oh, fuck! I didn't noticed it before, but all the modifications introduced to this file in the last two PRs should have been done to setup.cfg.tmpl instead!

File: templates/setup.cfg.tmpl [hunk 3]
```diff
@@ -10,8 +10,42 @@ parentdir_prefix = {{PROJECT_NAME}}
 addopts = --doctest-modules --cov=./tests
 
 [flake8]
-exclude = __init__.py,versioneer.py,{{SRC}}/_version.py
-max-line-length = 120
+exclude = __init__.py,versioneer.py,src/_version.py
```
Comment (NicolaDonelli) [inline templates/setup.cfg.tmpl:13]: In this line we must keep {{SRC}} instead of src

Comment (aiola) [reply templates/setup.cfg.tmpl]: Ops, sorry. Fixed

**Review by deusebio — APPROVED** [review]
Looks good!

**Pull request merged**